web/css/main.css.map

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This should also not be checked in…

Perhaps add those two files to the .gitignore?